spec/controllers/application_controller/feature_spec.rb

```diff
@@ -1,25 +1,21 @@
-require File.expand_path("../app/controllers/application_controller/", "feature.rb")
-
 describe ApplicationController do
```

Comment:
Can you just make this `describe ApplicationController::Feature do` and then use `described_class` below without having to get the constant off of it?